extensions/rest/src/main/java/org/radargun/traits/RESTOperations.java

```diff
@@ -0,0 +1,23 @@
+package org.radargun.traits;
+
+import java.util.List;
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.MultivaluedMap;
+import org.radargun.Operation;
+import org.radargun.WrappedHttpResponse;
+
+/**
+ * @author Martin Gencur
+ */
+@Trait(doc = "Http operations.")
+public interface RESTOperations {
+   String TRAIT = RESTOperations.class.getSimpleName();
+
+   Operation GET = Operation.register(TRAIT + ".Get");
+
+   RESTOperationInvoker getRESTInvoker(String contextPath);
```

Comment:
I think it is good enough to have the contextPath set at the beginning and then just re-used in the test. This is going to be used as a load test. It's not going to mimic Selenium which is able to go through several web pages and clicking on multiple buttons and requesting different URLs. Also, RG is not prepared for passing multiple different URLs and I'm not going to implement it now. We can add this option later when it's really needed. It will need a change in the configuration file anyway.